pkg/venafi/tpp/connector.go

```diff
@@ -582,6 +582,21 @@ func prepareRequest(req *certificate.Request, zone string) (tppReq certificateRe
 		tppReq.CASpecificAttributes = append(tppReq.CASpecificAttributes, nameValuePair{Name: expirationDateAttribute, Value: formattedExpirationDate})
 	}
 
+	if req.ValidityDuration != nil {
+		for _, attributeName := range []string{
+			"Microsoft CA:Specific End Date",
```

Comment:
The documentation for this attribute confuses me:
![image](https://github.com/jetstack/vcert/assets/978965/6b54fcda-44a5-4eaa-821a-84498c5cd98d)

> The exact certificate expiration date to use **the next time** the certificate renews.

Why **next time** ?

-- https://docs.venafi.com/Docs/currentSDK/TopNav/Content/SDK/WebSDK/SchemaReference/r-SDK-CA-Microsoft.php